android/guava-tests/test/com/google/common/base/FinalizableReferenceQueueClassLoaderUnloadingTest.java

```diff
@@ -55,7 +55,7 @@
    * to reference the FinalizableReferenceQueue class then its ClassLoader cannot be
    * garbage-collected, even if there are no more instances of FinalizableReferenceQueue itself.
    * The code in FinalizableReferenceQueue goes to considerable trouble to ensure that there are
-   * no such references and the tests here check that that trouble has not been in vain.
+   * no such references and the tests here check that trouble has not been in vain.
```

Comment:
This isn't an error. It's saying "check that X has not been in vain", where X is "that trouble [that the code goes to]".